app/apis/api_application.py

```diff
@@ -58,6 +63,14 @@ class ApiApplications(Resource):
     @app_info_namespace.marshal_list_with(app_info)
     def get(self):
         """get_applications"""
+        if Auth.enabled():
```

Comment:
Since `enabled()` method returns boolean value, `is_enabled()` will be better.

Reply:
fixed.